src/components/TextInput.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Field, propTypes } from 'redux-form';
+import { TextField } from 'redux-form-material-ui';
+
+
+const TextInput = ({ name, type, placeholder, ...props }) => (
+  <Field
+    name={name}
+    component={TextField}
+    type={type}
```

Comment:
I think it is referenced as **input.value: any** in the documentation where you can set the type of field. 
What i can see is in examples type is used extensively especially with **text**